docs/contributing/source/dev-setup-coding-and-contribution-process/run-dev-network-stack.md

```diff
@@ -0,0 +1,296 @@
+# Run a BigchainDB network
+
+**NOT for Production Use**
+
+You can use the following instructions to deploy a single or multi node
+BigchainDB network for dev/test using the extensible `stack` script(s). 
+
+Currently, this workflow is only supported for the following Operating systems:
+- Ubuntu >= 16.04
+- CentOS >= 7
+- Fedora >= 24
+- MacOSX
+
+## Minimum Requirements
+Minimum resource requirements for a single node BigchainDB dev setup. **The more the better**:
+- Memory >= 512MB
+- VCPUs >= 1
+
+## Download the scripts
+```text
+$ curl -fOL https://raw.githubusercontent.com/bigchaindb/bigchaindb/${GIT_BRANCH}/pkg/scripts/stack.sh
+
+# Optional
+$ curl -fOL https://raw.githubusercontent.com/bigchaindb/bigchaindb/${GIT_BRANCH}/pkg/scripts/unstack.sh
+```
+
+## Quick Start
+If you run `stack.sh` out of the box i.e. without any configuration changes, you will be able to deploy a 4 node
+BigchainDB network with Docker containers. 
```

Comment:
Maybe you should add that this is going to deploy a network where all four nodes are `Using bigchaindb branch 'master'` because if they're devs, they probably want to run _their_ code, not code from the `master` branch.

Reply:
I will, but if they are devs, `stack.sh` will clone a master, they can make changes in the cloned project and they can be reflected since we are using a `-e` flag while installing BigchainDB or they can simply `unstack.sh` and `stack.sh`

Reply:
I wouldn't have guessed that as the workflow for devs. Please explain that in the docs.